plugins/inputs/tail/README.md

```diff
@@ -67,6 +67,9 @@ The plugin expects messages in one of the
   ## Set the tag that will contain the path of the tailed file. If you don't want this tag, set it to an empty string.
   # path_tag = "path"
 
+  ## Remove ANSI colors
+  # remove_colors = false
+
```

Comment:
We try to avoid booleans in the config as it creates some stumbling blocks down the road if we need to expand on it. Maybe something like `remove = ["colors"]`?